src/text.rs

```diff
@@ -275,7 +363,7 @@ impl<'a> From<Spans<'a>> for String {
 /// ```
 #[derive(Debug, Clone, PartialEq, Default, Eq)]
 pub struct Text<'a> {
-    pub lines: Vec<Spans<'a>>,
+    pub lines: Vec<Line<'a>>,
```

Comment:
This line is the main line that makes this PR obviously the right thing to do.
We already refer to a Vec of Spans as `lines` :)

Reply:
Honestly, `Line` is a better name, until you render a paragraph with wrapping, then a `Line` can be several lines long.... (just an amusing observation that naming things is hard)

Reply:
I know this is probably a super pedantic take, but I'd still call that a line. It's a wrapped line, but still a line.